util/keyfile/Cargo.toml

```diff
@@ -48,5 +48,5 @@ x509-signature = "0.5"
 [dev-dependencies]
 mc-crypto-x509-test-vectors = { path = "../../crypto/x509/test-vectors" }
 mc-util-test-helper = { path = "../../util/test-helper" }
-
 tempfile = "3.2"
```

Comment:
IIUC we can now remove this.
```suggestion
```

Reply:
The bin-determinism test still depends on `tempfile::NamedTempFile`, so that needs to be replaced first